apps/site/src/views/Schedule/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,40 @@
+import { Variants, motion } from "framer-motion";
+import styles from "./Loader.module.scss";
+
+const variants: Variants = {
+	initial: {
+		scaleY: 0.5,
+		opacity: 0,
+	},
+	animate: {
+		scaleY: 1,
+		opacity: 1,
+		transition: {
+			repeat: Infinity,
+			repeatType: "mirror",
+			duration: 0.5,
+			ease: "circIn",
+		},
+	},
+};
+
+const BarLoader = () => {
```

Comment:
Nitpick: Since the component is called `BarLoader`, the file and its corresponding SCSS file should also be called `BarLoader` instead of `Loader`.